### DISCUSSION TITLE
npx nuxi upgrade --force: Current nuxt version does not get updated

### DISCUSSION THREAD

**Discussion by fabianwohlfart** [nuxt/nuxt#21024] [Questions]
Hi,

Does anybody know why the »Current nuxt version« does not update accordingly?

Version says 3.5.1 here
```
~: npx nuxi -v
Nuxi 3.5.1    
```

but here it's always rc.14?

```
~: npx nuxi@latest upgrade --force
Nuxi 3.5.1                                                                                                                                        11:31:23
                                                                                                                                                  11:31:23
ℹ Package Manager: pnpm 8.5.1                                                                                                                     11:31:23
ℹ Current nuxt version: 3.0.0-rc.14                                                                                                               11:31:23
ℹ Removing lock-file and node_modules...                                                                                                          11:31:23
ℹ Installing latest Nuxt 3 release...                
```

**Comment by hermanndettmann** [comment]
I'm having the same issue since version 3.5.1 and can't update my Nuxt app since then. Any ideas what to do to fix this?

**Reply by fabianwohlfart** [reply]
i don't use pnpm anymore, but yarn.

**Reply by hermanndettmann** [reply]
Oh I overlooked that detail. I'm actually using good ol' `npm`. :-)

**Comment by warflash** [comment]
Try removing package_lock and node_modules and reinstalling. If that doesn't fix it you might have installed rc14 installed globally or somewhere in a parent dir and it uses that instead, but not 100% sure on that atm

**Reply by hermanndettmann** [reply]
That didn't help either. I also don't have rc14 installed because I start here:
```
Package Manager: npm 8.19.4
Current nuxt version: 3.5.1
Removing lock-file and node_modules...
Installing latest Nuxt 3 release...     
```
and end up here:
```
Cleaning up generated nuxt files and caches...
ℹ Upgraded nuxt version: 3.5.1
✔ You're already using the latest version of nuxt.
```
when running `npx nuxi upgrade --force`.

**Comment by filipecruz** [comment]
I'm having the same issue, using a mac m1 with nvm for Node version selection, and have nuxt 3.10.3 define on package.json

Getting this after runnning: 'npx nuxi@latest upgrade --force'

 WARN  Module nuxt-auth-sanctum is disabled due to incompatibility issues:                                                                                                                                               2:24:56 PM
 - [nuxt] Nuxt version ^3.9.0 is required but currently using 3.6.1
                                                                                                

✔ Types generated in .nuxt                                                                                                                                                                                              2:24:57 PM
✨  Done in 42.31s.
ℹ Cleaning up generated nuxt files and caches...                                                                                                                                                                        2:24:57 PM
ℹ Upgraded nuxt version: 3.6.1                                                                                                                                                                                          2:24:57 PM
✔ You're already using the latest version of nuxt. 


And this after running 'npx nuxi info'

- Operating System: `Darwin`
- Node Version:     `v21.6.2`
- Nuxt Version:     `3.6.1`
- CLI Version:      `3.10.1`
- Nitro Version:    `2.5.2`
- Package Manager:  `yarn@1.22.19`
- Builder:          `-`
- User Config:      `devtools`, `runtimeConfig`, `build`, `modules`, `app`, `css`, `ssr`, `vite`, `components`, `eslint`, `typescript`, `svgo`, `router`, `vuetify`, `experimental`, `image`, `fontLoader`, `auth`
- Runtime Modules:  `@nuxtjs/eslint-module@4.1.0`, `@nuxt/image@1.3.0`, `nuxt-svgo@4.0.0`, `@pinia/nuxt@0.5.1`, `nuxt-build-cache@0.1.1`, `nuxt-font-loader@2.3.5`, `nuxt-auth-sanctum@0.1.2`, `nuxt-aos@1.2.3`, `()`
- Build Modules:    `-`

Anyone facing the same issue?
Regards

**Comment by richard-edwards** [comment]
I'm having the same issue. I currently have 3.10.3 and it won't upgrade to 3.11.x. I tried manually updating just the nuxt package to 3.11 and then ran the nuxi upgrade and it seems to have worked but now I'm getting another error:

```[8:35:59 PM] ERROR Nuxt Build Error: "joinRelativeURL" is not exported by "node_modules/ufo/dist/index.mjs", imported by "virtual:nuxt:/workspace/.nuxt/paths.mjs".```

**Comment by Rednas83** [comment]
For me it worked to update nuxt in **package.json** and then run `pnpm nuxi upgrade --force` in the terminal afterwords.

I think the nuxt version gets stuck after reverting an update manually.
